public/lib/helpers/timeline_helper.js

```diff
@@ -26,6 +30,28 @@ define(function (require) {
       }
     };
 
+    TimelineHelper.prototype.changeTimezone  = function ($scope) {
```

Comment:
Good idea. And this way the function we'll be easier to test since it would return a string.